pr-data.csv

```diff
@@ -2861,8 +2862,8 @@ https://github.com/IBM/og,bf82c95b488151e58d53e895509fce058065e044,og-configurat
 https://github.com/IBM/og,bf82c95b488151e58d53e895509fce058065e044,og-configuration,com.ibm.og.json.type.FilesizeConfigTypeAdapterFactoryTest.serialization,ID,Opened,https://github.com/IBM/og/pull/36,
 https://github.com/IBM/og,bf82c95b488151e58d53e895509fce058065e044,og-configuration,com.ibm.og.json.type.OperationConfigTypeAdapterFactoryTest.serialization,ID,Opened,https://github.com/IBM/og/pull/36,
 https://github.com/IBM/og,bf82c95b488151e58d53e895509fce058065e044,og-configuration,com.ibm.og.json.type.SelectionConfigTypeAdapterFactoryTest.serialization,ID,Opened,https://github.com/IBM/og/pull/36,
-https://github.com/IBM/platform-services-java-sdk,9397602ebd5329b035851542c84c3a347b23eb7d,modules/configuration-governance,com.ibm.cloud.platform_services.configuration_governance.v1.model.CreateRuleRequestTest.testCreateRuleRequest,ID,,,
-https://github.com/IBM/platform-services-java-sdk,9397602ebd5329b035851542c84c3a347b23eb7d,modules/configuration-governance,com.ibm.cloud.platform_services.configuration_governance.v1.model.RuleRequestTest.testRuleRequest,ID,,,
+https://github.com/IBM/platform-services-java-sdk,9397602ebd5329b035851542c84c3a347b23eb7d,modules/configuration-governance,com.ibm.cloud.platform_services.configuration_governance.v1.model.CreateRuleRequestTest.testCreateRuleRequest,ID,Opened,https://github.com/IBM/platform-services-java-sdk/pull/161,The maintainer don't want the code to be fixed since it's autogenerated
+https://github.com/IBM/platform-services-java-sdk,9397602ebd5329b035851542c84c3a347b23eb7d,modules/configuration-governance,com.ibm.cloud.platform_services.configuration_governance.v1.model.RuleRequestTest.testRuleRequest,ID,Opened,https://github.com/IBM/platform-services-java-sdk/pull/161,The maintainer don't want the code to be fixed since it's autogenerated
```

Comment:
This should be `Rejected`

Reply:
For this PR since it's not rejected due to our fixing but the maintainer doesn't want us to fix, should we still mark it as rejected?

Reply:
Yes, please put `Rejected` so that future students don't bother developers of this project again.